### DISCUSSION TITLE
Hiding source server IP

### DISCUSSION THREAD

**Discussion by eZwzAEkx** [postalserver/postal#1803] [Help with using Postal] [answered]
Hi,

I will want to use postal to send a message from CMS. However, I don't want to disclose the IP address of the server on which CMS is located.

Postal installation is on another server.

Is postal can be a source of disclose the IP address on which CMS is located?

**Comment by willpower232** [comment, marked as answer]
see #86 and maybe #1471

**Comment by eZwzAEkx** [comment]
Thank you Will. 

 And I apologize for the opening of the discussion without thoroughly checking.

**Reply by willpower232** [reply]
no worries :-)
